sdk/src/Core/Internal/Utils/EndPoint.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net;
```

Comment:
Lets add the copyright documentation to all new files.

Reply:
Addressed in 06a1695